docker/cloudtik-dev/Dockerfile

```diff
@@ -2,8 +2,12 @@
 # for developers that need the source code to actively modify.
 
 FROM cloudtik/cloudtik-deps:nightly
-ADD cloudtik.tar /cloudtik
-ADD git-rev /cloudtik/git-rev
+
+COPY cloudtik.tar /tmp/cloudtik.tar
+
+WORKDIR /cloudtik
+RUN tar -xf /tmp/cloudtik.tar
```

Comment:
Can we move and combine these two lines:
WORKDIR /cloudtik
RUN tar -xf /tmp/cloudtik.tar

to the:

WORKDIR /cloudtik/
RUN bash build.sh

instead of multiple RUN and repeat WORKDIR.

Reply:
Done.